src/app/shared/services/user.service.ts

```diff
@@ -22,6 +22,28 @@ export class UserService extends BaseBackendService<User> {
     super();
   }
 
+  public getApiKey(): Observable<string> {
+    return this.getList()
```

Comment:
Should we try to fetch single user instead of the whole list?
We can also use getList directly in the component and try to get user.apiKey there.
It can save us a single call